docs/guides/tutorials/nextjs_app_router.rst

```diff
@@ -528,9 +528,86 @@ you to ``/post/<uuid>``.
 Deploying to Vercel
 -------------------
 
+You can deploy an EdgeDB instance on the EdgeDB Cloud or 
+on your preferred cloud provider. We'll cover both options here.
+
+With EdgeDB Cloud
+=================
+
+**#1 Deploy EdgeDB**
+
+First, sign up for an account at `cloud.edgedb.com <https://cloud.edgedb.com>`_ 
+and create a new instance. Create and make note of a secret key for your EdgeDB Cloud instance. You 
+can create a new secret key from the "Secret Keys" tab in the EdgeDB Cloud 
+console. We'll need this later to connect to the database from Vercel.
+
+As we've already initialized a local project, we need to unlink it from the 
+local directory. Run the following command to unlink the project:
+
+.. code-block:: bash
+
+  $ edgedb project unlink
+
+Then run the following command to link the project to the EdgeDB Cloud:
+
+.. code-block:: bash
+
+  $ edgedb project init --server-instance "<org>/<instance-name>"
+
+The migrations and schema will be automatically uploaded to the cloud.
```

Comment:
I'm a little unsure about this as the recommended workflow. I would think the user might want to be able to continue to work locally, so I would probably recommend migrating via `edgedb migrate -I <org>/<instance-name>` and keeping the project link to the local instance.

Maybe we should get some input from @elprans and @1st1 on this to see what workflow we should recommend here…

Reply:
Ahh, I see. I thought that this is what you meant by "having to start from scratch".